### ISSUE TITLE
WCAG issue - The Pending tab in the Media Gallery is missing an ARIA label

### ISSUE THREAD

**Issue by Rainribbon** [Kaltura-EDU/issues-tracking#14]
Submitted as Kaltura Support Case #00548739:

When the Moderate Content setting is on, submissions to the Media Gallery that need to be approved or rejected by the instructor appear in the Pending tab. When using a screen reader, the tabs are described as "4 Media in Media Gallery <course ID>" or "Playlists in Media Gallery <course ID>". When the third Pending tab is in focus, the screen reader says, "In Media Gallery <course ID>." This does not provide the user with any context and the tab is missing an appropriate ARIA label, such as "Pending in Media Gallery 144437".

This issue does not meet the standards of [WCAG SC 2.4.6: Headings and Labels (Level AA)](https://www.w3.org/WAI/WCAG22/Understanding/headings-and-labels#dfn-label)
